src/bun.js/bindings/ErrorStackTrace.cpp

```diff
@@ -800,3 +800,8 @@ String functionName(JSC::VM& vm, JSC::JSGlobalObject* lexicalGlobalObject, const
     return functionName;
 }
 }
+
+extern "C" void Bun__errorInstance__finalize(void* bunErrorData)
```

Comment:
@Jarred-Sumner This dummy implementation is added for https://github.com/oven-sh/WebKit/commit/0875bc8f628e0015a1ad4e74efa5b89133c2e717, if you want to add finalization here please push commits to this branch